components/BabydegenEconomyPage/OptimusAgentMetrics.jsx

```diff
@@ -23,12 +23,16 @@ const fetchMetrics = async () => {
 };
 
 const formatNumber = (num) => {
+  if (num === null || num === undefined) return null;
   const numTo1dp = Number(num?.toFixed(1));
   return `${numTo1dp}%`;
 };
 
 export const OptimusAgentMetrics = () => {
-  const { data: metrics } = usePersistentSWR('OptimusMetrics', fetchMetrics);
+  const { data: metrics } = usePersistentSWR('OptimusMetrics', fetchMetrics, {
+    refreshInterval: 3600000, // Refresh every hour
+    revalidateOnFocus: true,
```

Comment:
I don't see this parameters are used in usePersistentSWR, will they work as expected?